perfzero/lib/perfzero/benchmark_method_runner.py

```diff
@@ -149,6 +155,8 @@ def _run_internal(benchmark_method, harness_info, site_package_info,
   report_utils.execute_methods(
       config.result_upload_methods,
       execution_summary)
+  _set_file_contents(json.dumps(execution_summary, indent=2),
+                     os.path.join(output_dir, 'perfzero_summary.json'))
```

Comment:
Currently the json-formatted summary will be printed to stdout and the perfzero.log file. Can you move the new logic to the same place in the code?

And can you update README and clarify that the json-formatted summary will be printed to this newly-added file? 

https://github.com/tensorflow/benchmarks/tree/master/perfzero#json-formatted-benchmark-summary

Reply:
Thanks, Updated the readme file.
I moved it L160 and L161. This is just after it's written to perfzero.log at L158.
Note that it's written to perfzero.log at the moment logging.info() is called at L158. 

The reason logging.info outputs to perfzero.log is due to the following code snippet at L77 above:
filehandler = logging.FileHandler(
      filename=os.path.join(output_dir, 'perfzero.log'), mode='w')
  filehandler.setFormatter(
      logging.Formatter('%(asctime)s %(levelname)s: %(message)s'))
  logging.getLogger().addHandler(filehandler)